metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -700,6 +711,16 @@ class TableauSourceReport(
         default_factory=(lambda: defaultdict(int))
     )
 
+    get_connection_objects_query_counter: Dict[str, int] = dataclass_field(
+        default_factory=(lambda: defaultdict(int))
```

Comment:
could you also add a comment explaining that the keys are limited to a constant set, which is why we don't need to use a LossyDict here

Reply:
addressed in commit ab10931